### PR TITLE
blog(december): update template file for december posts

### PR DESCRIPTION
**Pull request by jayjayjpg**
This adds a few more details to the guidelines for December blog posts and also adds a ready-to-copy template file for contributors to get started.

If you find this change useful, feel free to let me know if this can be further improved or if you'd expect it to be in a different format!

### CODE REVIEW

File: source/CONTRIBUTING.md [hunk 1]
```diff
@@ -134,10 +134,14 @@ git commit -m "you-commit-message-here"
...
+- Create a pull request from your repo on Github.
 
-There should be an open pull request titled: `[WIP] The Ember Times No. #` Please include reference to this PR in your pull request. 
+There should be an open pull request titled: `[WIP] The Ember Times No. #` Please include reference to this PR in your pull request.
 
 For example: "Add a section to Ember Times [`#46`](https://github.com/ember-learn/ember-blog/pull/46)"
 
 Thank you for contributing to Ember Times!
+
+## December
```
Comment (MelSumner) [inline source/CONTRIBUTING.md]: DecEmber :)

**Review by MelSumner — APPROVED** [review]
:100: Thank you Jessica!

File: source/december-2019-template.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+title: Countdown to The New Year - XYour Addon's Name In Title CaseX
```
Comment (amyrlam) [inline source/december-2019-template.md:2]: lgtm! 

nit - feel free to ignore - been thinking about this re: Ember Times ... I'm beginning to wonder if it makes more sense to list addons in lowercase as they are on github e.g. `ember-svg-jar` or `prettier`

File: source/CONTRIBUTING.md (hunk 1 above)
Comment (jayjayjpg) [inline source/CONTRIBUTING.md]:
```suggestion
## DecEmber
```

Comment (jayjayjpg) [reply source/CONTRIBUTING.md]: Sounds great, updated :-)

File: source/december-2019-template.md (hunk 2 above)
Comment (jayjayjpg) [reply source/december-2019-template.md:2]: Yeah I agree as well! I think in a previous PR we discussed that [in the scope of the December series, we'd like to title case in the actual title of the post, but use the lowercase version everywhere else](https://github.com/ember-learn/ember-blog/pull/377#discussion_r351007857) - does this make sense and what are your thoughts?

Comment (amyrlam) [reply source/december-2019-template.md:2]: @jessica-jordan 👍 sgtm, thanks for the context

**Pull request merged**